app/main.py

```diff
@@ -1,3 +1,42 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km: float = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance") -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: "Distance") -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        if isinstance(other, (int, float)):
+            self.km += other
+        return self
+
+    def __mul__(self, scalar: float) -> "Distance":
+        return Distance(self.km * scalar)
+
+    def __truediv__(self, scalar: float) -> "Distance":
+        return Distance(round(self.km / scalar, 2))
+
+    def __lt__(self, other: "Distance") -> bool:
+        return self.km < other
+
+    def __gt__(self, other: "Distance") -> bool:
+        return self.km > other
+
+    def __eq__(self, other: "Distance") -> bool:
+        return self.km == other
+
+    def __le__(self, other: "Distance") -> bool:
+        return self.km <= other
+
+    def __ge__(self, other: "Distance") -> bool:
+        return self.km >= other
```

Comment:
![image](https://github.com/mate-academy/py-distance-class/assets/103201010/7a1cea01-0586-4f9a-acc7-fb8a6de71f60)
1. no need to use quotes in type annotation 
2. other could be an int and distance class
3. will now code work with distance class correctly ?